src/assets/locales/en.json

```diff
@@ -240,6 +240,12 @@
   "home.terms_of_use": "Terms of Use",
   "home.made_with_love": "made with 🦆 by ",
   "home.environment_author_by": " by ",
+  "home.rooms-title": "Instantly create rooms",
+  "home.rooms-blurb": "Share virtual spaces with your friends, co-workers, and communities. When you create a room with Hubs, you’ll have a private virtual meeting space that you can instantly share - no downloads or VR headset necessary.",
+  "home.communicate-title": "Communicate naturally",
+  "home.communicate-blurb": "Choose an avatar to represent you, put on your headphones, and jump right in. Hubs makes it easy to stay connected with voice and text chat to other people in your private room.",
```

Comment:
I'm a bit concerned that the "put on your headset" part will turn non-VR users away, but it may be OK because `rooms-blurb` says "no downloads or VR headset necessary"

Reply:
lol, yep. :facepalm:

Reply:
John are you referring to the "put on your headphones" part? The 'No VR headset necessary' phrase was to try and alleviate concerns about needing a headset when we use the terms 'social VR', so we definitely want to make that clear.